packages/react-experiments/src/components/FloatingSuggestions/FloatingSuggestions.tsx

```diff
@@ -175,13 +175,13 @@ export class FloatingSuggestions<TItem>
     // If it is then resolve it asynchronously.
     if (Array.isArray(suggestions)) {
       this.updateSuggestions(suggestions, true /*forceUpdate*/);
-    } else if (suggestions && suggestions.then) {
+    } else if (suggestions && (suggestions as PromiseLike<TItem[]>).then) {
```

Comment:
This change and the several similar ones were prompted by stricter uncalled function checks flagging the `.then`